app/views/worldwide_organisations/show.html.erb

```diff
@@ -1,100 +0,0 @@
-    <p class="govuk-body-l worldwide-org-summary"><%= @worldwide_organisation.summary %></p>
-    <div class="worldwide-org-description">
-      <% if @draft %>
-        <%= govspeak_edition_to_html @worldwide_organisation.draft_about_us %>
-      <% else %>
-        <%= govspeak_edition_to_html @worldwide_organisation.about_us %>
-      <% end %>
-    </div>
-  </div>
-  <% if @worldwide_organisation.social_media_accounts.any? %>
-    <aside class="social-media-links govuk-grid-column-one-third">
-      <div class="content">
-        <h2 class="govuk-heading-s"><%= t("worldwide_organisation.headings.follow_us") %></h2>
-        <%= render "govuk_publishing_components/components/share_links", {
-          track_as_follow: true,
-          links: @worldwide_organisation.social_media_accounts.map do |account|
-            {
-              href: account.url,
-              text: account.display_name,
-              icon: account.service_name.parameterize,
-            }
-          end
-        } %>
-      </div>
-    </aside>
-  <% end %>
-</section>
-
-<% if ([@primary_role]+@other_roles).compact.any?(&:has_appointment?) %>
-  <section class="govuk-grid-row" id="people">
-    <div class="govuk-grid-column-full">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: t("worldwide_organisation.headings.our_people"),
-        border_top: 5,
-        padding: true,
-      } %>
-    </div>
-    <%= render( partial: "people/person",
-        locals: {
-          person: @primary_role.current_person,
-          roles: [@primary_role],
-          hlevel: "h3",
-          wrapping_element: :div,
-          extra_class: "govuk-grid-column-one-quarter",
-        }
-      ) if @primary_role %>
-    <div class="govuk-grid-column-three-quarters">
-      <ul class="govuk-list govuk-grid-row">
-        <% @other_roles.each.with_index do |role, i| %>
-          <%= render( partial: "people/person",
-            locals: {
-              person: role.current_person,
-              roles: [role],
-              hlevel: "h3",
-              hide_image: true,
-              extra_class: (i + 1) % 4 == 0 ? "govuk-grid-column-one-third clear-column" : "govuk-grid-column-one-third",
-            }
-          ) %>
-        <% end %>
-      </ul>
-    </div>
-  </section>
-<% end %>
-
-<% if @main_office %>
-  <section id="contact-us">
-    <%= render "govuk_publishing_components/components/heading", {
-      text: t("worldwide_organisation.headings.contact_us"),
-      border_top: 5,
-      padding: true,
-    } %>
-    <%= render partial: "contacts/organisation_contact", locals: {
-      contact: @main_office,
-      is_main: true,
-      lang: t_lang_translated_locales(@main_office.contact)
-    } %>
-    <% @home_page_offices.each do |home_page_office| %>
-      <%= render partial: "contacts/organisation_contact", locals: {
-        contact: home_page_office,
-        lang: t_lang_translated_locales(home_page_office.contact)
```

Comment:
I _think_ the `t_lang_translated_locales` method can now be removed, since it appears not to be used elsewhere (other than in it's own test).

Reply:
It's gone!